components/MoodPicker/MoodPickerItem.vue

```diff
@@ -14,8 +14,8 @@
       <div class="mood-picker-item-back p-4 flex flex-col justify-between">
         <div>
```

Comment:
do we need both divs (l. 15 & l. 22)?
my suggestion would be deleting both, adding py-2 & px-4 to the parent div, removing mb-2 from all child-elements.

Reply:
This used to make sense when the boxes were higher. And the to divs were pushed apart because of `justify-between`. Right now, I question the existence of the second div with the "back" button which has the flipBox click handler. Maybe we can put the click handler directly onto the backside?